.gitignore

```diff
@@ -28,3 +28,6 @@ yarn-error.log*
 .env.development.local
 .env.test.local
 .env.production.local
+
+# WebStorm configs
+.idea
```

Comment:
Github is complaining of no new line, might not be a bad idea to add in.

Reply:
AFAIK it doesn't matter in this case, but it doesn't hurt. I'll update my IDE to auto add newlines on save. I have that setting on my work machine, but didn't have it set on my personal machine for some reason.